PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/uihelper/InteractiveRenderer.java

```diff
@@ -35,16 +35,8 @@ public InteractiveRenderer(int interactiveColumn) {
     @Override
     public Component getTableCellRendererComponent(JTable table, Object value, boolean isSelected, boolean hasFocus, int row, int column) {
         Component component =  super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
-        if (column == interactiveColumn && hasFocus) {
-
-            InteractiveTableModel tableModel = (InteractiveTableModel)table.getModel();
-            if(tableModel != null)
-
-            if ((tableModel.getRowCount() - 1) == row && !tableModel.hasEmptyRow())
-            {
-               tableModel.addEmptyRow();
-            }
```

Comment:
I think one empty row is needed, so users can double click this row to add their customized row